lib/govuk_app_config/govuk_data_sync.rb

```diff
@@ -0,0 +1,31 @@
+require "time"
+
+class GovukDataSync
+  def initialize(govuk_data_sync_period)
+    @from, @to = false
+    begin
+      parts = govuk_data_sync_period&.split("-")
+      @from, @to = parts.map { |time| Time.parse(time) } if parts&.count == 2
+    rescue ArgumentError
+      # At least one of the parts was malformed. Leave @from/@to as false.
+    end
+  end
+
+  def in_progress?
+    @from && @to && in_time_range?(@from, @to)
```

Comment:
Similar to previous comment about attr_reader being the more common idiom in GOV.UK ruby